src/server/game/Entities/Player/Player.cpp

```diff
@@ -1081,6 +1081,11 @@ void Player::Update(uint32 p_time)
 
     UpdateAfkReport(now);
 
+    if (IsInCombat())
```

Comment:
@Treeston does your PR #19930 provide anything for pvp combat ? :)
I see that there is ```bool HasPvPCombat()``` does it do what I need here ?